lib/commands/touch.js

```diff
@@ -0,0 +1,124 @@
+import _ from 'lodash';
+import androidHelpers from '../android-helpers';
+import { asyncmap } from 'asyncbox';
+
+let commands = {}, helpers = {}, extensions = {};
+
+helpers.parseTouch = async function (gestures, multi) {
```

Comment:
if this logic is the same as from appium-android-driver, we should definitely export it there and import it here rather than duplicating it.